detectors/node/opentelemetry-resource-detector-aws/README.md

```diff
@@ -7,7 +7,11 @@
 
 Resource detector for Amazon Web Services.
 
-Compatible with OpenTelemetry JS API and SDK `1.0+`.
+## Status
+
+| Maturity                                  | Maintainer                                             | Compatibility         |
+| ----------------------------------------- | ------------------------------------------------------ | --------------------- |
+| [Stable](../../../CONTRIBUTING.md#stable) | [@carolabadeer](../../../.github/component_owners.yml) | API 1.0+<br/>SDK 1.0+ |
```

Comment:
The link of the GitHub handle might be superfluous since it is not linked to a specific person rather than a list of owners. If we have several maintainers listed here, I don't think it would be helpful to link them all.

```suggestion
| [Stable](../../../CONTRIBUTING.md#stable) | @carolabadeer | API 1.0+<br/>SDK 1.0+ |
```

Reply:
What about linking the header?

```md
| Maturity                                  | [Component Owner](../../.github/component_owners.yml) | Compatibility         |
| ----------------------------------------- | ----------------------------------------------------- | --------------------- |
| [Stable](../../../CONTRIBUTING.md#stable) | @carolabadeer                                         | API 1.0+<br/>SDK 1.0+ |
```